test/models/task_list_test.rb

```diff
@@ -3,4 +3,33 @@
 require 'test_helper'
 
 class TaskListTest < ActiveSupport::TestCase
+  test 'should save model' do
+    test_task = TaskList.new(name: 'Clean up')
+    assert test_task.valid?
+  end
+
+  test 'task list name too long' do
+    test_task = TaskList.new(name: '0123456789101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899')
```

Comment:
For this case, `SecureRandom.hex(number_of_chars)` is your friend:
https://ruby-doc.org/stdlib-2.7.0/libdoc/securerandom/rdoc/SecureRandom.html#module-SecureRandom-label-Examples